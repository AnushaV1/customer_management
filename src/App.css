.App {
  text-align: center;
}

.form-control {
  text-align: start;
  margin-bottom: 0;
}

.error {
  color: red;
  text-align: start;
  
}

table {
  max-width: 100%;
  border: none;
  border-spacing: 0;
  text-align: left;
  overflow-x: hidden;
}
th, tr {
width:10%;
}