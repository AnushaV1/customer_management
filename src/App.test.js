import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import App from '../src/App'

it('renders without crashing', function () {
	render(
			<BrowserRouter>
				<App />
			</BrowserRouter>

	);
});

it('matches snapshot', function () {
	const { asFragment } = render(
			<BrowserRouter>
				<App />
			</BrowserRouter>

	);
	expect(asFragment()).toMatchSnapshot();
});
