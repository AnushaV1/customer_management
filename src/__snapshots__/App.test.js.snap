// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <nav
        class="navbar navbar-expand-md navbar-dark bg-dark"
      >
        <div
          class="container-fluid"
        >
          <div
            class="navbar-header"
          >
            <a
              aria-current="page"
              class="navbar-brand active"
              href="/"
            >
              Customer Management
            </a>
          </div>
          <ul
            class="nav navbar-nav navbar-right"
          >
            <li
              class="nav-item"
            >
              <a
                href="/newcustomers"
              >
                Add New Customer
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                href="/listcustomers"
              >
                List Customers
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
    <div>
      <section
        class="Home col-md-12 p-0"
      >
        <div
          class="banner container-fluid text-center text-light m-0 p-5"
        >
          <div
            class="content-wrapper mb-5"
          >
            <p
              class="lead"
            >
              Customer Management in one convenient place
            </p>
            <a
              class="btn btn-primary btn-lg px-5 py-3 my-5"
              href="/newcustomers"
            >
              Add New Customer
            </a>
             
            <a
              class="btn btn-primary btn-lg px-5 py-3 my-5"
              href="/listcustomers"
            >
              List Customers
            </a>
          </div>
        </div>
      </section>
    </div>
  </div>
</DocumentFragment>
`;
